README.md

```diff
@@ -121,6 +150,18 @@ YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ MASTER=yarn-client ./sparkR
 YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ ./sparkR examples/pi.R yarn-client
 ```
 
+### Using sparkR-submit
+sparkR-submit is a script introduced to faciliate submission of SparkR jobs to a YARN cluster.
+It supports the same commandline parameters as [spark-submit](http://spark.apache.org/docs/latest/running-on-yarn.html). SPARK_HOME, YARN_HOME, and JAVA_HOME must be defined.
```

Comment:
Actually we should link to http://spark.apache.org/docs/latest/submitting-applications.html -- And it'll also be good to work in the fact that sparkR-submit can be used to submit jobs to any cluster manager (similar to spark-submit)